script/data/load_dataset.sh

```diff
@@ -0,0 +1,13 @@
+
+#!/bin/sh
+
+# name of the datset
+DATASET_NAME=$1
+
+# get script path
+SCRIPT_PATH=$(dirname $0)
+
+echo "Loading dataset ${DATASET_NAME} into EVA"
+
+# execute the python script
+python $SCRIPT_PATH/load_dataset.py ${DATASET_NAME}
```

Comment:
Is load_dataset supposed to be the part of this PR?

Reply:
No. The merge with other PR didn't go well. I actually removed it. Let me see what is going on

Reply:
I don't have these files on my machine